client/src/components/Topics/index.js

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import './style.css';
 
-export function Topics() {
+function Topics(props) {
     return (
         <div>
             <div className="topics-title">
                 <h1>Hot Topics</h1>
             </div>
             <div className="top-topics">
+                <ul>
+                    {props.post.map(item => (
+                        <p key={item.id}>
```

Comment:
`<div>` should be used instead of `<p>` to fix the react errors

Reply:
i added the new commit